src/utils/__tests__/options.test.js

```diff
@@ -48,6 +48,18 @@ describe('options', () => {
     expect(toDependency(nameWithVersion)).toEqual({ name: nameWithVersion });
     nameWithVersion = `@packageName@${version}`;
     expect(toDependency(nameWithVersion)).toEqual({ name, version });
+    // Version with @ symbol
+    expect(toDependency(`packageName@file:local/@my-packages`)).toEqual({
```

Comment:
I found this example slightly confusing, but as explained irl, its looking at an unscoped package name with a path that contains an '@'. Super unlikely to ever happen, but an important code path to double check that the logic is correct.